### PR TITLE
Add the graviton usage

### PR DESCRIPTION
**Pull request by athiruma**
## Type of change
Note: Fill **x** in []
- [ ] bug
- [ ] enhancement
- [x] documentation
- [] dependencies

## Description
Add the graviton usage report to ES

### CODE REVIEW

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:1]: Where is the api for fetching graviton usage ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:1]: The payer account activated the tag: architecture.
By using this tag we can get the cost of graviton instance usage.

File: jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py [hunk 2]
```diff
@@ -24,19 +23,22 @@
...
-os.system(f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="perf-scale" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index={es_index_perf_scale} -e cost_explorer_tags="{cost_tags}" -e granularity={granularity} -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+os.system(
+    f"""podman run --rm --name cloud-governance -e AWS_DEFAULT_REGION="us-east-1" -e account="perf-dept" -e policy="cost_explorer" -e AWS_ACCESS_KEY_ID="{AWS_ACCESS_KEY_ID_DELETE_PERF}" -e AWS_SECRET_ACCESS_KEY="{AWS_SECRET_ACCESS_KEY_DELETE_PERF}" -e es_host="{ES_HOST}" -e es_port="{ES_PORT}" -e es_index="{es_index_perf}" -e cost_explorer_tags="{cost_tags}" -e granularity="{granularity}" -e cost_metric="{cost_metric}" -e log_level="INFO" quay.io/ebattat/cloud-governance:latest""")
+os.system(
```
Comment (ebattat) [inline jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:38]: Why we need to change podman run ?

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:38]: No, I prettified the code according to the IDE suggestions.

Comment (athiruma) [reply jenkins/clouds/aws/daily/cost_explorer/run_upload_es.py:38]: yes, expect the `architecture` tag

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**